Draft-Accepted/Install-PowerShell.md

```diff
@@ -0,0 +1,217 @@
+---
+RFC: RFC00XX
+Author: Jason Helmick
+Status: Draft
+SupercededBy: N/A
+Version: 1.0
+Area: Core
+Comments Due: 6/30/2022
+Plan to implement: Yes
+---
+
+# Install-PowerShell7 Cmdlet for Windows
+
+This RFC proposes the addition of the `Install-PowerShell7` cmdlet in Windows to perform a
+simplified installation of the latest stable release of PowerShell 7. Options through parameters are
+provided for advanced installation scenarios.
+
+## Motivation
+
+Many IT customers of PowerShell face a problem in which they don't control the landscape of software
+pre-installed on the Windows Servers they manage. Explicitly installing PowerShell is a blocker for
+some customers due to discoverability and some that have policies that don't allow for installing
+software onto Windows after initial provisioning.
+
+To compensate for these situations, customers will implement and use the "tools at hand" to perform
+management tasks. This leaves **Windows PowerShell 5.1**, not **PowerShell 7**, as the only
+automation solution available to many of them. While customers are missing out on useful new
+features that they may wish to have, they are certainly not receiving the performance and security
+improvements that come with each new release.
+
+This specification is an effort to help IT admins that wish to use new and updated releases of
+PowerShell on Windows and to easily install PowerShell from **Windows PowerShell 5.1** with a
+cmdlet. While this may not help customers that have post-deployment policy restrictions, it will
+help the majority of customers discover and update to the latest version of PowerShell for Windows.
+
+```
+As an admin,
+I can use Windows PowerShell 5.1 to provision multiple hosts with PowerShell 7 over PSRemoting,
+so that I can leverage new capabilities in PS7.
+```
+
+```
+As a user,
+I can discover and install PowerShell 7 from Windows PowerShell 5.1 on my Windows machine,
+so that I can leverage new capabilities in PS7.
+```
+
+## Shipping in Windows
+
+There are several issues that prevent shipping PowerShell 7 in Windows;
+
+- Size
+- Support lifecycle differences
+
+### Size constraints
+
+ A complete installation of PowerShell 7 currently is around 200 MB (unpacked). This is due to
+ including everything (modules and .NET assemblies) to provide parity with what Windows PowerShell
+ 5.1 with .NET Framework provides. There is currently no plan by Windows or the .NET team to ship
+ newer .NET in Windows, so PS7 will need to continue to ship .NET with PS7.
+
+### Support lifecycle
+
+ Windows has a 5+5 year support lifecycle. .NET supports up to 3 years for Long Term Support (LTS)
+ versions. So even if size was not an issue, we cannot check in a version of PS7 with a .NET that is
+ not supported while Windows itself is supported.
+
+### Solution
+
+To solve these issues, the proposal is to ship a new cmdlet in Windows and the PowerShell Gallery
```

Comment:
Would that be a new built-in module in Windows?

Reply:
Yes -- I'll add. PowerShell7 is the working title of the module - we can change.